_shared/project/conf/supervisord-dev.conf

```diff
@@ -1,29 +1,32 @@
 [supervisord]
 nodaemon = true
 silent = true
-{% if cookiecutter.get("_directory") == "pyapp" %}
 
-[program:app]
-command=python3 -m {{ cookiecutter.package_name }}
-stdout_events_enabled=true
-stderr_events_enabled=true
-stopsignal = KILL
-stopasgroup = true
+{% if include_exists("supervisor.json") %}
+{% set programs = include_json("supervisor.json").get("programs", {}) %}
+{% else %}
+{% if cookiecutter._directory == 'pyramid-app' %}
+{% set programs = {"web": {"dev": true, "prod": true, "settings": {"command": "gunicorn --bind :{{ cookiecutter.port }} --workers 1 --reload --timeout 0 --paste conf/development.ini"}}} %}
+{% else %}
+{% set programs = {"app": {"dev": true, "prod": true, "settings": {"command": "python3 -m h_periodic"}}} %}
```

Comment:
The default programs for non-web apps that don't have a `supervisor.json` file. Whoops, need to not hard-code `h_periodic` here.